movies_mvc/movies_mvc/Core/Api/Url.swift

```diff
@@ -0,0 +1,11 @@
+// Url.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Апи
+enum Url {
```

Comment:
```suggestion
enum Url {
```
лучше не назвать системными словами которые есть в foundation